### PR TITLE
fix(TBD-11309): fix bigdata launcher issue for hdi 3.6

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by CarlosTalend — APPROVED** [review]

File: main/plugins/org.talend.hadoop.distribution.hdinsight360/plugin.xml [hunk 1]
```diff
@@ -133,13 +133,12 @@
...
 
-        <!-- Big Data launcher group -->
+         <!-- Big Data launcher group -->
         <libraryNeededGroup
                 description="Big Data launcher libraries for HDInsight 3.6.0"
                 id="BIGDATA-LAUNCHER-LIB-HD_INSIGHT_3_6_0"
                 name="BIGDATA-LAUNCHER-LIB-HD_INSIGHT_3_6_0"  >
-            <library id="talend-bigdata-launcher"/>
-            <library id="talend-bigdata-launcher-hdinsight"/>
+	            <library id="talend-bigdata-launcher-hdinsight"/>
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.hdinsight360/plugin.xml:141]: why this change if there's no such PR for maintenance/7.3? What was tested OK didn't contain this change

**Review by CarlosTalend — CHANGES_REQUESTED** [review]
Needs same modifications as maintenance/7.3

File: main/plugins/org.talend.hadoop.distribution.hdinsight360/plugin.xml (hunk 1 above)
Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.hdinsight360/plugin.xml:147]: Thanks Carlos, Modified that.

Comment (CarlosTalend) [reply main/plugins/org.talend.hadoop.distribution.hdinsight360/plugin.xml:147]: thanks, Sveta

**Review by CarlosTalend — APPROVED** [review]
